siphon/simplewebservice/iem.py

```diff
@@ -0,0 +1,106 @@
+""" Requests data from IEM asos.py"""
+
+import requests
+import datetime
+import pandas as pd
+
+from io import StringIO
+
+from ..http_util import create_http_session
+
+
+class IemAsos:
```

Comment:
Hmmm. `IEMAsos`, `'IEMasos`, `IAStateASOS`, `IAStateasos`? Not sure on fighting convention for readability. Thoughts @dopplershift ?